### ISSUE TITLE
terminate called after throwing an instance of 'std::out_of_range'

### ISSUE THREAD

**Issue by mwuertinger** [iovisor/bpftrace#1005]
```
#!/usr/bin/env -S bpftrace --unsafe
  
tracepoint:syscalls:sys_enter_open,
tracepoint:syscalls:sys_enter_openat
/args->flags!=0 && str(args->filename)=="/some/path"/ {
        system("echo foo");
}
```
This script produces the following error message:
```
terminate called after throwing an instance of 'std::out_of_range'
  what():  vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)
Aborted
```

Removing the `system()` or the filter fixes the problem. Removing either of the two probes leads to the following error:
```
libclang: crash detected during parsing: {
  'source_filename' : 'definitions.h'
  'command_line_args' : ['clang', '-isystem', '/usr/local/include', '-isystem', '/bpftrace/include', '-isystem', '/usr/include', '-nostdinc', '-isystem', '/virtual/lib/clang/include', '-I/lib/modules/5.0.0-37-generic/build/arch/x86/include', '-I/lib/modules/5.0.0-37-generic/build/arch/x86/include/generated', '-I/lib/modules/5.0.0-37-generic/build/include', '-I/lib/modules/5.0.0-37-generic/build/include', '-I/lib/modules/5.0.0-37-generic/build/arch/x86/include/uapi', '-I/lib/modules/5.0.0-37-generic/build/arch/x86/include/generated/uapi', '-I/lib/modules/5.0.0-37-generic/build/include/uapi', '-I/lib/modules/5.0.0-37-generic/build/include/generated/uapi', '-include', '/lib/modules/5.0.0-37-generic/build/include/linux/kconfig.h', '-D__KERNEL__', '-D__BPF_TRACING__', '-D__HAVE_BUILTIN_BSWAP16__', '-D__HAVE_BUILTIN_BSWAP32__', '-D__HAVE_BUILTIN_BSWAP64__', '-DKBUILD_MODNAME='"bpftrace"'', '-include', 'asm_goto_workaround.h'],
  'unsaved_files' : [('definitions.h', '...', 276), ('/bpftrace/include/__stddef_max_align_t.h', '...', 1771), ('/bpftrace/include/float.h', '...', 5192), ('/bpftrace/include/limits.h', '...', 3735), ('/bpftrace/include/stdarg.h', '...', 2025), ('/bpftrace/include/stddef.h', '...', 4499), ('/bpftrace/include/stdint.h', '...', 23388), ('/bpftrace/include/asm_goto_workaround.h', '...', 667)],
  'options' : 1,
}
```
System information:
```
 $ bpftrace --version
bpftrace v0.9.2-102-g254b
$ uname -a
Linux pc-540 5.0.0-37-generic #40-Ubuntu SMP Thu Nov 14 00:14:01 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=19.04
DISTRIB_CODENAME=disco
DISTRIB_DESCRIPTION="Ubuntu 19.04"
```

**Comment by fbs** [comment]
Pretty sure this has been fixed with #959 , can you try with master?

**Comment by mwuertinger** [comment]
It indeed works with master. Thanks!
